packages/react-app/src/components/pages/Bounties/Form/NewBounty.tsx

```diff
@@ -0,0 +1,249 @@
+import { Controller, useForm } from 'react-hook-form';
+import React, { useContext } from 'react';
+import { useRouter } from 'next/router';
+import DatePicker from '@app/components/parts/DatePicker';
+import {
+	Box,
+	FormLabel,
+	FormControl,
+	Input,
+	Button,
+	FormErrorMessage,
+	Select,
+	Flex,
+	FormControlProps,
+	Textarea,
+} from '@chakra-ui/react';
+import { BountyCollection } from '@app/models/Bounty';
+import bountyStatus from '@app/constants/bountyStatus';
+import { CustomerContext } from '@app/context/CustomerContext';
+import { APIUser } from 'discord-api-types';
+import { useUser } from '@app/hooks/useUser';
+import { useLocalStorage } from '@app/hooks/useLocalStorage';
+import { dateIsNotInPast, required, validNonNegativeDecimal } from '@app/utils/formUtils';
+
+const PLACEHOLDERS = {
+	TITLE: 'Example: Create new Logo',
+	DESCRIPTION: 'Example: We need someone to create some snappy looking logos for our new Web3 project.',
+	CRITERIA: 'Example: SVG and PNG images approved by the team',
+};
+
+const useCurrencies = (): string[] => {
+	return ['BANK'];
```

Comment:
We'll want a multitenant approach for this in the future with pulling the allowlisted currencies from a customer config, but for now, here are the currencies we allow on the bot:

```
const ALLOWED_CURRENCIES = ['BANK', 'ETH', 'BTC', 'USDC', 'USDT', 'TempCity', 'gOHM', 'LUSD', 'FOX', 'oneFOX', 'DIMO', '$DIMO'];
```

Reply:
Leaving multitenant implementation of create out of scope for this release